src/Compilers/Shared/RuntimeHostInfo.cs

```diff
@@ -29,6 +32,11 @@ internal static (string processFilePath, string commandLineArguments, string too
         {
             Debug.Assert(!toolFilePathWithoutExtension.EndsWith(".dll") && !toolFilePathWithoutExtension.EndsWith(".exe"));
 
+            var standaloneToolFilePath = $"{toolFilePathWithoutExtension}{StandaloneToolSuffix}";
+            if (IsCoreClrRuntime && File.Exists(standaloneToolFilePath))
```

Comment:
Small comment on the choice of "standalone" as a name prefix here. 

This will eventually impact more than standalone projects. The default starting in `netcoreapp3.1`, maybe `netcoreapp3.0`, is to produce a native executable beside the tool `dll`. For example the build of `VBCSCompiler.csproj` should natively produce `VBCSCompiler.dll` and `VBCSCompiler.exe` on windows.

That is disabled for the moment via https://github.com/dotnet/roslyn/commit/5184394d4bbea910f1a37b34f1e4295952334907. Eventually though we are going to start producing these native app host binaries again. Probably as a function of building our installers in the https://github.com/dotnet/core-sdk repository. The native app host binaries are advantages because it means in the process list you see `VBCSCompiler.exe` not dozens of `dotnet.exe` processes. 

That being said this is the right change to make. It will work for both the standalone case as well as the native app host one. The name "standalone" is specific to one of them though. That being said I can't think of a better name to use here. 😕

Reply:
Maybe `NativeToolFilePath`?

Reply:
That works for me'